e2e-tests/tests/__fixtures__/hmr/updated-button-body.tsx

```diff
@@ -9,9 +9,7 @@ export const initialProps: ButtonProps = {
 export default function Button(props: ButtonProps) {
   return (
     <button className={props.className}>
-      <div className="bg-purple-400 text-white">
```

Comment:
the bg-purple class no longer works since we're not including the full tailwind bundle on the page